backend/src/assets/404.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Page not found | Sleep API</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta property="og:url" content="https://sleepapi.net" />
```

Comment:
Can you remove this one too, I don't want any thumbnail for 404 pages and I think this would still cause a barebones thumbnail to appear on social media shares